### ISSUE TITLE
Arcade - World.pause() + World.step() = weird behavior

### ISSUE THREAD

**Issue by jjcapellan** [photonstorm/phaser#4972]
## Version

* Phaser Version: 3.22.0

* Operating system: Debian GNU / Linux 10 (Kernel 4.19.0-6-amd64)

## Description  
Within a scene, if we pause the physics engine in the create() and advance it step by step in the update(), using the <code>scene.physics.world.pause()</code> and <code>scene.physics.world.step(delta)</code> respectively. The following occurs in moving objects:
1.- The gameObject moves away from its body.
2.- The gameObject and the body have different display origins after being created.
It seems that this problem or similar appeared in version 3.17.

## Example Test Code

Code forked of @samme [3.15 example](https://codepen.io/samme/pen/vbgxeL), using version 3.22.0:  
https://codepen.io/jjcapellan/pen/qBEwVwK

## Additional Information

This is the chain that causes this problem:
1. World.isPaused == true
2. World.update() is not executed
3. Body.preUpdate() is not executed
4. Body.prevFrame is never updated and origins not synchronized
5. In Body.postUpdate():
```javascript
postUpdate: function ()
    {
        // this.prevFrame vector is never updated
        var dx = this.position.x - this.prevFrame.x;
        var dy = this.position.y - this.prevFrame.y;

        // Some code here ...
            
            this.gameObject.x += dx;
            this.gameObject.y += dy;
        }
       // Some code here ...
   }
```
This is so far my workarround without touching Phaser:
```javascript
    create(){
        this.physics.world.pause();
    }

    update(time, delta){
        this.preUpdateBodies();
        this.physics.world.step(0.001 * delta);
    }

    preUpdateBodies(){
        let body;
        let bodies = this.physics.world.bodies.entries;

        for (let i = 0; i < bodies.length; i++)
        {
            body = bodies[i];

            this.preUpdateBody(body);
        }
    }

    preUpdateBody(body){
        body.resetFlags();
        body.updateBounds();

        var sprite = body.transform;

        body.position.x = sprite.x + sprite.scaleX * (body.offset.x - sprite.displayOriginX);
        body.position.y = sprite.y + sprite.scaleY * (body.offset.y - sprite.displayOriginY);

        body.updateCenter();

        body.rotation = sprite.rotation;

        body.preRotation = body.rotation;

        if (body.moves)
        {
            body.prev.x = body.position.x;
            body.prev.y = body.position.y;
            body.prevFrame.x = body.position.x;
            body.prevFrame.y = body.position.y;
        }
    }
```
Maybe adding some flag to detect when the world.step() function has been used ...  
Regards.

**Comment by photonstorm** [comment]
This issue has been mentioned on **Phaser**. There might be relevant details there:

https://phaser.discourse.group/t/phaser-3-flappy-tours-2/4877/4

**Comment by jjcapellan** [comment]
Problem solved in my pull request #4989
